addons/crm/views/crm_lead_views.xml

```diff
@@ -258,8 +258,10 @@
                                 <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color', 'no_create_edit': True}"/>
```

Comment:
Commit message typos:

- when the user execute -> executes
- the button become -> the button becomes
- until the user refresh -> refreshes
- this button use -> uses
- unnecessary -> unnecessarily

Reply:
(Can you also update the PR description? In case we use rebase-merge for this)